Samples/WinMLSamplesGallery/WinMLSamplesGallery/Common/File.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Runtime.InteropServices;
+using Windows.Storage.Pickers;
+using Windows.Storage;
+using Microsoft.UI.Xaml;
+using Windows.Graphics.Imaging;
+using Windows.Media;
+using Microsoft.UI.Xaml.Media.Imaging;
+
+namespace WinMLSamplesGallery.Common
+{
+    [ComImport, Guid("3E68D4BD-7135-4D10-8018-9FB6D9F33FA1"), InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    public interface IInitializeWithWindow
+    {
+        void Initialize([In] IntPtr hwnd);
+    }
+
+    public static class File

```

Comment:
"File" feels too generic for the name of this class.

Reply:
same with the name of this file, "File.cs". also, there's an image helper later in this file, so not sure if "File.cs" accurately describes what's in here.

Reply:
Changed to ImageHelper